ToDoApp/Backend/src/db/data.js

```diff
@@ -0,0 +1,30 @@
+let todos = [{
+  id: '1',
+  message: 'first todo',
+  finished: false,
+  publishedBy: '1'
+},
+  {
+    id: 2,
+    message: 'second todo',
+    finished: true,
+    publishedBy: '2'
+  },
+];
+
+let users = [{
+  id: '1',
+  name: 'eva testuser',
+  email: 'your@email.com',
+  password: '$2y$10$3IUx11G0mcJfSpFWn3Lru.xac9OqHDzqLOAhdZovaUyKa2DhgCaOS' // "password"
+},
+  {
+    id: '2',
+    name: 'anna testuser',
+    email: '2your@email.com',
+    password: '$2y$10$3IUx11G0mcJfSpFWn3Lru.xac9OqHDzqLOAhdZovaUyKa2DhgCaOS' // "password"
+  }
+]
+
+module.exports.todos = todos;
+module.exports.users = users;
```

Comment:
I put in this PR for another group to show you how to setup babel. https://github.com/m-oehme/Systems-Development-and-Frameworks/pull/6 With babel you can use `import` and `export` statements.